<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnggDiffractionQtTabGSAS</class>
 <widget class="QWidget" name="EnggDiffractionQtTabGSAS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_generalParams">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string>General Parameters</string>
   </property>
   <widget class="QLabel" name="label_runFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Nexus file containing a spectrum focused using EnggFocus or the Focus tab</string>
    </property>
    <property name="text">
     <string>Focused Run File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_runFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browseRunFile">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadRun">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_instParamsFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>GSAS-style instrument file (extension *.prm)</string>
    </property>
    <property name="text">
     <string>Instrument Parameter File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_instParamsFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browseInstParams">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phaseFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>*.cif files containing the phases to refine</string>
    </property>
    <property name="text">
     <string>Phase Files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phaseFiles">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browsePhaseFiles">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gsasProjPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The name of a new *.gpx project to write refinement results out to. This can be opened and used for more complex refinements in GSAS-II</string>
    </property>
    <property name="text">
     <string>New GSAS-II Project</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gsasProjPath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gsasProjPath">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gsasHome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Directory containing GSAS-II installation. This directory must contain GSASIIscriptable.py</string>
    </property>
    <property name="text">
     <string>GSAS-II Installation Directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gsasHome">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browseGSASHome">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_refinementMethod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Rietveld or Pawley</string>
    </property>
    <property name="text">
     <string>Refinement Method</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_refinementMethod">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pawley</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rietveld</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_pawleyParams">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Pawley Parameters</string>
   </property>
   <widget class="QLabel" name="label_pawleyDMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The minimum d-spacing to use. Leave empty to use the default (1)</string>
    </property>
    <property name="text">
     <string>DMin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pawleyDMin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_pawleyNegativeWeight">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The weight of the penalty function for negative intensities. Leave empty to use default (0)</string>
    </property>
    <property name="text">
     <string>Negative Weight</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pawleyNegativeWeight">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_fitResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>641</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Fit Results</string>
   </property>
   <widget class="QwtPlot" name="plotArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>4</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>2</width>
      <height>2</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showRefinementResults">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Overplot fitted peaks and display fit output parameters, if a fit has been run</string>
    </property>
    <property name="text">
     <string>Plot Refinement Results</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_runLabels">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>75</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_latticeParams">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>20</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Lattice Parameters</string>
    </property>
    <widget class="QLabel" name="label_latticeParamA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_latticeParamB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_latticeParamC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_latticeParamAlpha">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>alpha</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_latticeParamBeta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>beta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_latticeParamGamma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>gamma</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_latticeParamA">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_latticeParamB">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_latticeParamC">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_latticeParamAlpha">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_latticeParamBeta">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_latticeParamGamma">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_rwp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Weight profile R-factor, goodness of fit index</string>
    </property>
    <property name="text">
     <string>Rwp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rwp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
